src/api/proto/vispb/vis.proto

```diff
@@ -308,6 +308,13 @@ message TimeseriesChart {
   Axis y_axis = 4;
 }
 
+message TextChart {
+  message Text {
```

Comment:
I did this because we were doing it elsewhere with similarly simple+singleton fields. I should have checked if there was a reason for that; I can simplify this if preferred.

Reply:
Simplified.